app/_variables.scss

```diff
@@ -24,14 +24,14 @@ $light-navy: #13327a;
 $sapphire: #5030a5;
 
 // Size Variables
-$navbar-height: 60px;
-$container-width: 1180px;
-$small-container-width: 780px;
-$search-container-width: 895px;
-$mobile-width: 768px;
+$navbar_height: 60px;
+$container_width: 1180px;
+$small_container_width: 780px;
+$search_container_width: 895px;
+$mobile_width: 768px;
 
 // s3 url
-$s3-url: "https://dd2gn9pwu61vr.cloudfront.net";
+$s3_url: "https://dd2gn9pwu61vr.cloudfront.net";
```

Comment:
We can use assets.pluto.network

Reply:
That is the address of pluto landing page asset S3.

Reply:
Then we should make the normalized address later.